cmd/upgrade/executor/cstor_spc.go

```diff
@@ -103,6 +103,18 @@ func (u *UpgradeOptions) RunCStorSPCUpgrade(cmd *cobra.Command) error {
 			fmt.Println(err)
 			return errors.Errorf("Failed to upgrade cStor SPC %v:", u.cstorSPC.spcName)
 		}
+	case "1.1.0-1.2.0":
```

Comment:
Can we use a multiple case statement like: 
```
case "1.0.0-1.1.0", "1.0.0-1.2.0", "1.1.0-1.2.0":
```

Example: https://github.com/golang/go/wiki/Switch

Reply:
Done